### ISSUE TITLE
Color leaf in diffrent color

### ISSUE THREAD

**Issue by Or-Geva** [jpb12/react-tree-graph#85]
Can I color some nodes in the tree? Set only the leaf to red, for example.

**Comment by jpb12** [comment]
https://jpb12.github.io/react-tree-graph/?path=/story/tree-nodes--custom-node-props

**Comment by Or-Geva** [comment]
Thanks for the super fast response @jpb12 !. I try to set the leaves in a different className and see it is working.
However, the rest of the nodes get undefined for their className. shouldn't the default className suppose to be applied?

**Comment by jpb12** [comment]
Can you share the relevant code?

**Comment by Or-Geva** [comment]
Here is my animatedTree:
```
		<AnimatedTree
				data={root}
				height={props.height * 0.95}
				width={props.width}
				keyProp="id"
				gProps={{
					className: 'node',
					onClick: handleClick
				}}
				pathProps={{
					className: 'path lin' `
				}}
				textProps={{
					dy: 3.5
				}}
				steps={30}/>
```

To set the root in red, for example, I would do the following
```
root.gProps = {className: 'redNode'}
```

As a result, `onClick` will not work on the root node, but will work on the rest.

Moreover, I encounter [this](https://github.com/jpb12/tree-viewer/blob/master/Components/treeContainer.js#L55) example, but it is not working.

**Comment by jpb12** [comment]
Should be fixed in 8.0.1

**Comment by Or-Geva** [comment]
Thanks!
